ui/src/shared/components/TimeMachine/CSVExporter.tsx

```diff
@@ -2,15 +2,28 @@ import React, {PureComponent} from 'react'
 import {connect} from 'react-redux'
 
 import {Button, IconFont, ComponentStatus} from 'src/reusable_ui'
-import downloadTimeseriesCSV from 'src/shared/utils/downloadTimeseriesCSV'
+import {
+  downloadInfluxQLCSV,
+  downloadFluxCSV,
+} from 'src/shared/utils/downloadTimeseriesCSV'
 import {notify} from 'src/shared/actions/notifications'
-import {csvExportFailed} from 'src/shared/copy/notifications'
+import {
+  csvExportFailed,
+  fluxResponseTruncatedError,
+} from 'src/shared/copy/notifications'
 
-import {Query, Template} from 'src/types'
+import {Query, Template, Service} from 'src/types'
 
 interface Props {
+  // Used for downloading an InfluxQL query
```

Comment:
what do you think about making Props a union / extension of `InfluxQLQueryProps` and `FluxQueryProps`?